client/src/components/chess-board/ChessBoard.js

```diff
@@ -1,11 +1,95 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import './ChessBoard.css';
 import ChessSquare from '../chess-square/ChessSquare';
 
-const ChessBoard = () => (
-  <div>
-    ChessBoard Component
-    <ChessSquare />
-  </div>
-);
+const matchingStringInArray = array => str => array.some(x => x.includes(str));
+
+class ChessBoard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
create them as class variables so that you don't have to use the constructor syntax

Reply:
Done